### PR TITLE
Saba/lab tests functionality

### PR DESCRIPTION
**Pull request by SabaSaleem5317**
- Added a lab test page.
labtest
- Updated backend and frontend URLs to connect to AWS.
- Added a .env file in the backend for database connection.
- Modified Dockerfile by removing versions of some modules.
- Created a Results page where users can view and download their results; currently displaying dummy data.

https://github.com/user-attachments/assets/cf0c3b45-b0c6-4119-b9be-3b764c89c2f5

- When Ans will complete admin portal then I'll Add results to S3 via the admin portal allowing users to retrieve them through the Results Page.

### CODE REVIEW

**Review by anssidhu — APPROVED** [review]
Looks good

**Review by muhammadqasim21 — APPROVED** [review]

File: frontend/src/App.js [hunk 1]
```diff
@@ -26,19 +27,20 @@ function App() {
   return (
     <Router>
       <div className="app">
-        <Routes>
-          <Route path="/" element={!showForm ? <LandingPage onSignIn={handleSignIn} onSignUp={handleSignUp} /> : (
+         <Routes>
+          {/* <Route path="/" element={!showForm ? <LandingPage onSignIn={handleSignIn} onSignUp={handleSignUp} /> : (
```
Comment (UrwaIhtesham) [inline frontend/src/App.js]: you commented out this part. was it intentional?

File: frontend/src/Components/Login/LoginForm.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ const LoginForm = ({ mode }) => {
 
     try {
       if (mode === 'login') {
-        const response = await axios.post('http://localhost:5000/login', { email: username, password });
```
Comment (UrwaIhtesham) [inline frontend/src/Components/Login/LoginForm.js:48]: is the backend deployed wroking properly for now?

File: frontend/src/results/results.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
```
Comment (UrwaIhtesham) [inline frontend/src/results/results.js:1]: is it responsive yet?

**Review by UrwaIhtesham — APPROVED** [review]

**Review by SabaSaleem5317 — COMMENTED** [review]
Change results.pdf
Remove classes from results.css that are not being used

File: frontend/src/results/results.js (hunk 3 above)
Comment (SabaSaleem5317) [reply frontend/src/results/results.js:1]: yesss

File: frontend/src/Components/Login/LoginForm.js (hunk 2 above)
Comment (SabaSaleem5317) [reply frontend/src/Components/Login/LoginForm.js:48]: yasss

File: frontend/src/App.js (hunk 1 above)
Comment (SabaSaleem5317) [reply frontend/src/App.js]: yes I commented it intentionally because I was working on Results page only

File: backend/app/__init__.py [hunk 4]
```diff
@@ -3,17 +3,27 @@
...
+# Configure the SQLAlchemy database URI
+    app.config['SQLALCHEMY_DATABASE_URI'] = (
+    f"mysql+pymysql://{db_username}:{db_password}@"
+    f"{db_url}/{db_name}"
+         )
     db.init_app(app)
     bcrypt.init_app(app)
     jwt.init_app(app)
-    CORS(app, resources={r"/*": {"origins": "http://localhost:3000"}})  # Allow CORS for your frontend URL
+    CORS(app, resources={r"/*": {"origins": ["http://localhost:3000", "http://animalia-frontend-bucket.s3-website-us-east-1.amazonaws.com"]}})
```
Comment (azubair-experlabs) [inline backend/app/__init__.py:27]: Nice

File: frontend/.env [hunk 5]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=http://http://44.204.83.159/:5000
```
Comment (azubair-experlabs) [inline frontend/.env:1]: Url format seems incorrect here

**Review by azubair-experlabs — APPROVED** [review]

File: frontend/.env (hunk 5 above)
Comment (SabaSaleem5317) [reply frontend/.env:1]: ok I'll check

**Review by NoorFatyma — APPROVED** [review]

**Pull request closed without merging**